pkg/runner/cluster_k8s.go

```diff
@@ -256,33 +256,35 @@ func (c *ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow io.W
 		return nil, err
 	}
 
-	var gg errgroup.Group
+	if input.TotalInstances <= 500 {
```

Comment:
Obviously this value is up for discussion, or we should flat-out remove this part.

Reply:
I would give a :+1: for just removing this section as well.